addons/sys_gestures/XEH_postInit.sqf

```diff
@@ -0,0 +1,29 @@
+#include "script_component.hpp"
+if (is3DEN || {!hasInterface || {!(GVAR(gesturesEnabled))}}) exitWith {};
```

Comment:
```suggestion
if (is3DEN || {!hasInterface || {!(GVAR(enabled))}}) exitWith {};
```
And elsewhere.

Reply:
Actually seems to already be done as a setting, so this was probably a bug. :P

Reply:
Hold over from old non-ACRE code!